### ISSUE TITLE
UI tweaks for mobile menu

### ISSUE THREAD

**Issue by mbyczkowski** [mbyczkowski/lacocina#27]
After #26 is merged, we need to touch up some of the styles (pending @abyczkowska's designs).

**Comment by katiemthom** [comment]
@abyczkowska Do you have the designs for these tweaks already?

**Comment by abyczkowska** [comment]
@katiemthom pretty much what's in zeplin (so the gray color is different, more padding and line spacing increase)

**Comment by mbyczkowski** [comment]
Currently, the mobile menu looks like this:
<img width="355" alt="screen shot 2016-09-17 at 9 24 11 pm" src="https://cloud.githubusercontent.com/assets/888422/18612887/3e334ae4-7d1d-11e6-8331-0d9267db973b.png">

Design:
<img width="379" alt="screen shot 2016-09-17 at 9 26 06 pm" src="https://cloud.githubusercontent.com/assets/888422/18612892/6fef77d8-7d1d-11e6-8ccd-86490004945e.png">

zeplink link (mac app): zpl://screen?pid=57709d1c2774e92f0cbf5b16&sid=57a686c9a07dce3d791f6218

**Comment by mbyczkowski** [comment]
This is done. If there are more designs coming we'll open a new issue.
